Client/Frontend/Browser/URLBarView.swift

```diff
@@ -81,6 +81,7 @@ class URLBarView: UIView {
 
     var toolbarIsShowing = false
     var topTabsIsShowing = false
+    var newTabUserResearch: NewTabUserResearch?
```

Comment:
added user research variable so that we get accurate results for whether to show or hide reload button in url bar